website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+    - Display parent folder name for files with same name in Changes section
+    - Navigate the new IDE features quickly using [the IDE User Interface](/docs/cloud/dbt-cloud-ide/ide-user-interface) help page
+    - Use `top X` in SQL when previewing in the IDE
+    - Opt into the new IDE backend layer over the past month (still with dbt-rpc). Ready for beta later in June!
+
+    ## Product refinements 
+
+    - Performance-related upgrades:
+        - Reduced cold start time by 60+%
+        - Improved render time of modals in the IDE by 98%
+        - Improved IDE performance with dbt Core v1.5+ (faster and snappier – highly encourage you to [upgrade your dbt version](/docs/dbt-versions/upgrade-dbt-version-in-cloud)!)
+    - Upgraded sqlfmt (which powers the Format button) to 0.18.0
+    - Updated Build button to change menu options based on file/model type (snapshot, macro, etc.)
+    - Display message to disable adblocker for file contents error
+    - Moved Format button to console bar
+    - Made many security enhancements in the IDE
+
+    ## Bug fixes
+
+    - File icon sizes no longer get wonky in small screen
+    - Toast notifications no longer take over command bar menu
+    - Hover info inside the text editor no longer gets cut off
+    - Transition between a file and a recently modified scratchpad no longer triggers a console error
+    - dbt v1.5+ now can access the IDE
+    - Confirm button on the Unsaved Changes modal now closes after clicking it
+    - Long node names no longer overflow in the parsed logs section in history drawer
+    - Status pill in history drawer no longer scales with longer command
+    - Tooltip for tab name with a long file name is no longer cut off
+    - Lint button should no longer available in main branch
+
+  </expandable>
+
+- <expandable alt_header='Run history improvements'>
+
+    New usability and design improvements to the **Run History** dashboard in dbt Cloud are now available. These updates allow people to discover the information they need more easily by reducing the number of clicks, surfacing more relevant information, keeping people in flow state, and designing the look and feel to be more intuitive to use.   
+
+    <Lightbox src="/img/docs/release-notes/run-history-improvements.gif" title="Improvements to Run History in dbt Cloud" />
+
+    Highlights include:
+
+    - Usability improvements for CI runs with hyperlinks to the branch, PR, and commit SHA, along with more discoverable temporary schema names
+    - Preview of runs' error messages on hover
+    - Hyperlinks to the environment
+    - Better iconography on run status
+    - Clearer run trigger cause (API, scheduled, pull request, triggered by user)
+    - More details on the schedule time on hover
+    - Run timeout visibility
+
+    dbt Labs is making a change to the metadata retrieval policy for Run History in dbt Cloud. 
```

Comment:
might be a good to have a separator so it reads better and its clear they're slightly diff updates. e.g. **Run history endpoint updates**

https://github.com/dbt-labs/docs.getdbt.com/pull/5510/files#diff-0a1bbc2ae4c8e2a68d5954dd5b178a7566d52ce592e945c8b5746a245bbd91ccL1-L7

Reply:
or make its own entry since the next entry is a duplicate it seems

Reply:
See above - this was just a move, so if it's in here twice, it was in the legacy arrangement that way as well.